app/services/edge_cache/buster/nginx.rb

```diff
@@ -0,0 +1,24 @@
+module EdgeCache
+  class Buster
+    class Nginx
+      def self.call(path)
+        uri = URI.parse("#{ApplicationConfig['OPENRESTY_URL']}#{path}")
+        http = Net::HTTP.new(uri.host, uri.port)
+        response = http.request Net::HTTP::Purge.new(uri.request_uri)
+
+        raise StandardError, "Nginx Purge request failed: #{response.body}" unless response.is_a?(Net::HTTPSuccess)
+
+        response.body
+      end
+    end
+  end
+end
+
+# Creates our own purge method for an HTTP request,
+# which is used by Nginx to bust a cache.
+# See Net::HTTPGenericRequest for attributes/methods.
+class Net::HTTP::Purge < Net::HTTPRequest # rubocop:disable Style/ClassAndModuleChildren
```

Comment:
I find it a bit weird to have this defined here and given that we had to disable a Rubocop warning for this I'm probably not alone. I'd say this should be in `app/lib`.